docs/API_ref.md

```diff
@@ -1,87 +1,26 @@
 <h1>API References</h1>
 
-### [DataProtectionApplicationSpec](https://pkg.go.dev/github.com/openshift/oadp-operator@master/api/v1alpha1#DataProtectionApplicationSpec)
+Pre-requisites: Install OADP to your cluster. before proceeding to the next steps.
```

Comment:
What about listing the CRDs that this will install? So they know what they should be looking for. 
"Once the operator is installed, the following CRDs will be available:...."

Reply:
@shubham-pampattiwar where was I
> ... giving the instructions to install the CRDs manually

([comment](https://github.com/openshift/oadp-operator/pull/990#pullrequestreview-1398549854))
in https://github.com/openshift/oadp-operator/commit/42a72811bd48048e582c64af90c1fa964d2b45fa ?